mender-convert

```diff
@@ -133,6 +133,11 @@ Examples for expert actions:
 EOF
 }
 
+show_version() {
+  local version=$(git describe --tags --dirty --exact-match 2>/dev/null || git rev-parse --short HEAD)
```

Comment:
I would write it like this:
```
local version=$(cd "$(dirname "$0")" && ( git describe --tags --dirty --exact-match 2>/dev/null || git rev-parse --short HEAD ) )
```
This way it will work also if called from outside of the folder.

Reply:
Right. Done.